README.adoc

```diff
@@ -75,12 +75,44 @@ brew tap advancedtelematic/otaconnect
 brew install aktualizr
 ----
 
+You can build and install the latest development version of aktualizr on MacOS (current head of the development branch):
+----
+brew tap advancedtelematic/otaconnect
+brew install --HEAD aktualizr
+----
+
+If any of the previous release versions of aktualizr has been installed before make sure you `unlink` it before
+installing the HEAD version
```

Comment:
Tiny, but:
```suggestion
installing the HEAD version:
```
And actually, the linebreak between "before" and "installing" is unnecessary.